content/cloud-docs/api-docs/providers.mdx

```diff
@@ -50,15 +50,18 @@ For publicly curated providers, the Terraform Cloud Registry acts as a proxy to
 - The public registry discovery endpoints have the path prefix provided in the [discovery document](/registry/api-docs#service-discovery) which is currently `/api/registry/public/v1`.
 - [Authentication](/cloud-docs/api-docs/#authentication) is handled the same as all other Terraform Cloud endpoints.
 
-## List public Terraform Registry Providers for an Organization
+## List Public Terraform Registry Providers for an Organization
 
 `GET /organizations/:organization_name/registry-providers`
 
+### Parameters
+
 | Parameter            | Description                                                    |
 | -------------------- | -------------------------------------------------------------- |
 | `:organization_name` | The name of the organization to list available providers from. |
 
-Lists the providers from the public Terraform Registry that included in the private registry for the specified organization.
+
+Lists the public and private providers  included in the private registry for the specified organization. 
```

Comment:
The title here suggests this is just public, but there we say both public and private - should we change one?